src/ImageSharp/Configuration.cs

```diff
@@ -83,6 +83,11 @@ public Configuration(params IConfigurationModule[] configurationModules)
         /// </summary>
         public IEnumerable<IImageFormat> ImageFormats => this.imageFormats;
 
+        /// <summary>
+        /// Gets or sets the position in a stream to use for reading.
+        /// </summary>
+        public ReadOrigin ReadOrigin { get; set; } = ReadOrigin.Begin;
```

Comment:
I don't think we should make this the default. We should not seek within the stream unless the user instructs us to do so.

Reply:
Based on my normal expectations I would be inclined to think the same. However, I think many other devs would expect it to JustWork™

Reply:
I personally think this is a bad decision but you are the "Grand High Eternal Dictator" so you have the final say. I do think we need an option for this and it is a great idea to add this but this should not be the default.

Reply:
Haha! We'll see what the other chaps think. Off to bed now so will come back to it in the morning.

Reply:
I'm not convinced at all that we need to do this, specially not by default!
See my comment on #477.
Or am I missing something? 😕